pkg/resource/processing_job/testdata/test_suite.yaml

```diff
@@ -1,17 +1,174 @@
-tests:
+  tests:
   - name: "Processing job create tests"
     description: "Part of processing job CRD tests."
```

Comment:
Edit this description to be more meaningful and in sync with the other tests here